.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: generic
+
+matrix:
+  include:
+    - name: 'Conda Python with .yml-file'
+      install:
+        - sudo apt-get update && sudo apt-get install -y wget
+        - wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh;
+        - bash miniconda.sh -b -p $HOME/miniconda
+        - source "$HOME/miniconda/etc/profile.d/conda.sh"
+        - hash -r
+        - conda config --set always_yes yes --set changeps1 no
+        - conda update -q conda
+        - conda create --name data_pipeline_api --file environment.yml
+        - conda activate data_pipeline_api
+      script:
+        - pytest ./data_pipeline_api/test_api.py
```

Comment:
I think we can just add the upload conda script after this. It will build a noarch package. So that should just work fine. This is a bit different than it's usually done with travis, I think, but it should work for us. I think in most projects the release is triggered by pushing a version tag to github. However, I think integrating it like that is a little more involved.

The commit sha will still be inside the conda package, so that should be all we need for now in terms of being trackable. It'd be nice if we could auto-tag here somehow, but I think that'd require some git credentials.

Reply:
Also, you'll probably need to install conda-build before running the script.

Reply:
We'll need a separate issue to build via CI.

Reply:
Issue to build via CI https://github.com/ScottishCovidResponse/SCRCIssueTracking/issues/532